contrib/storage-kafka/src/main/java/org/apache/drill/exec/store/kafka/schema/KafkaMessageSchema.java

```diff
@@ -76,11 +75,10 @@ public Table getTable(String tableName) {
     if (tableNames == null) {
       try (KafkaConsumer<?, ?> kafkaConsumer = new KafkaConsumer<>(plugin.getConfig().getKafkaConsumerProps())) {
         tableNames = kafkaConsumer.listTopics().keySet();
-      } catch(KafkaException e) {
-        throw UserException.dataReadError(e).message("Failed to get tables information").addContext(e.getMessage())
-            .build(logger);
+      } catch (Exception e) {
+        logger.warn("Failed to get tables information: {}", e.getMessage());
       }
     }
-    return tableNames;
+    return tableNames != null ? tableNames : Collections.emptySet();
```

Comment:
`tableNames` can be `null` only for the case when the exception was caught. Is it makes sense to add return statement there and revert the change in this line?